### DISCUSSION TITLE
A GPT dedicated to AIR/AS3: ActionScript DieHard

### DISCUSSION THREAD

**Discussion by ciacob** [airsdk/Adobe-Runtime-Support#3028] [General]
Hello everyone,

I took my chance in starting up a GPT that is dedicated to everything AS3 and gave it as good a start as I could. I would need you to play with it and come back with issues and omissions you find and ideas on how to make it better.

You need to have a paid ChatGPT subscription to use it, I'm afraid. Money goes to OpenAI, not to me :-)

Here's the direct link: https://chat.openai.com/g/g-m5WW0lo4b-actionscript-diehard

Or use the [Explore GPTs](https://chat.openai.com/gpts) feature, and search for __ActionScript DieHard__.

Thank you,
Claudius

**Comment by Ender22** [comment]
That's awesome! I'm going to give it a try.
Is it also trained on Starling and Feathers?

**Reply by ajwfrost** [reply]
It sounds great - I don't have a paid subscription myself but I know we have as a company so I'll see if I can get hold of the access rights.. 

Am wondering how all this works .. because surely things like the Bing AI chat feature will be able to pull in all the general web contents (stackoverflow, Adobe dev documentation etc) and spew out answers relating to AS3/etc? Is this a custom thing that has specific focus/training? Wondering what sort of questions you're thinking it would handle?

thanks

**Comment by ciacob** [comment]
I am yet to find myself how this exactly works, or how effective it really is. 

When configuring the **ActionScript DieHard**, I was taken to a sort of console, where the "classic" GPT walked me through a sort of free-form questionnaire, where I described what I want from this dedicated GPT I was building. There were both very practical questions, such as name, icon, problem domain and purposes, but also behavior tweaks, such as, how would the new GPT behave when generating code, what its tone of voice would be, etc.

I also added a number of web resources that are relevant to contemporary AS3 development, and I got back something of a "promise" that they will be used to great extent.

I still have an "Edit" button on that **ActionScript DieHard** GPT, where I am allowed to inject notes, observations and requirements into the model -- in fact, this is why I am asking people to use it and tell me what's less than ideal there: because I (think I) am able to address that.

I used it myself for a couple hours, and I found it rather helpful. But I need more feed-back. This is work in progress...
